src/Allocation.jl

```diff
@@ -24,10 +24,8 @@ include("lib/datastore.jl")
 
     # Extracted water (1000 m3) to be set by optimisation - super source represents failure.
     waterfromgw = Parameter(index=[regions, time], unit="1000 m^3")
-    waterfromreservoir = Parameter(index=[regions,time], unit="1000 m^3")
```

Comment:
The water that comes from the reservoirs is already counted in swsupply. Do we need to know if it comes from a reservoir ? If yes, do we define waterfromreservoir as the withdrawals at the gauges that are (linked to) reservoirs ? Or do we want something more sophisticated in the style of what exceeds what can be provided by the "natural" flow = withdrawals-(inflows-outflows) ?